examples/custom-datagrid-column/toolpad/components/LabelColumn.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
```

Comment:
As this is an example would it be better if it used `value`, `row` and `field` so that it shows everything that can be done and also the typings people should use?
But this could be improved later too.

Reply:
I feel like using `row` would be the advanced use-case. Personally I'd rather leave it out.

Reply:
Ok, just a thought, i think that's fine too as it probably represents the most common use case.